### PR TITLE
Home Style fixes (bunch of)

### LINKED ISSUE #219: Home Hero: fix text new lines
Screenshot 2023-11-01 at 10 51 04

### LINKED ISSUE #222: Header mobile close button size and position
Screenshot 2023-11-01 at 11 03 05

### LINKED ISSUE #223: Header: reposition middle links in original position
This will increase MLR approval first; then we can optimize for SEO
Screenshot 2023-11-01 at 11 21 22

### LINKED ISSUE #224: Home: first paragraph width
Screenshot 2023-11-01 at 11 29 30

### LINKED ISSUE #227: Next: fix middle text
Screenshot 2023-11-01 at 11 55 52

### LINKED ISSUE #231: Accordion 2 title sizes
Screenshot 2023-11-01 at 12 16 37

### PR DESCRIPTION
**Pull request by paolomoz**
Fix #219
Fix #222 
Fix #223 
Fix #224
Fix #227
Fix #231 
- Before: https://main--bevespi--hlxsites.hlx.page/
- After: https://fixes-01-11-2--bevespi--hlxsites.hlx.page/

### CODE REVIEW

**Comment by lansami** [comment]
For #223 on very small devices, the links are not displayed, not sure if is intended because I see the rule `header .nav-middle > div > div > div div:nth-of-type(2) p:nth-of-type(4)` applied for devices with max-width
on Bevespi they are displayed

**Comment by lansami** [comment]
Other than the comment above, all looks good

File: blocks/header/header.css [hunk 1]
```diff
@@ -197,31 +203,45 @@ header .nav-middle > div > div > div {
...
+}
 }
 
 header .nav-middle > div > div > div div:nth-of-type(2) p:nth-of-type(-n+2) {
   display: none;
 }
 
 header .nav-middle > div > div > div div:nth-of-type(2) p:nth-of-type(3) {
-  right: 72px;
+  /* right: 72px; */
```
Comment (lansami) [inline blocks/header/header.css:239]: commented code should be removed

File: blocks/header/header.css [hunk 2]
```diff
@@ -197,31 +203,45 @@ header .nav-middle > div > div > div {
...
 }
 
 header .nav-middle > div > div > div div:nth-of-type(2) p:nth-of-type(4) {
-  right: 14px;
+  /* right: 14px; */
```
Comment (lansami) [inline blocks/header/header.css:243]: commented code should be removed

**Comment by paolomoz** [comment]
> For #223 on very small devices, the links are not displayed, not sure if is intended because I see the rule `header .nav-middle > div > div > div div:nth-of-type(2) p:nth-of-type(4)` applied for devices with max-width on Bevespi they are displayed

it is intended, also because the original sites looks broken on very small width (see image)
I hope this is not breaking the MLR process :) 

Screenshot 2023-11-01 at 11 59 20

**Pull request merged**